webcompat/templates/issue/upload-image.jst

```diff
@@ -1,19 +1,25 @@
 <script type="text/template">
-  <form>
-    <label class="wc-UploadForm wc-UploadForm--new" for="image">
-      <div class="wc-UploadForm-wrapper">
-        <div class="wc-UploadForm-label wc-UploadForm-label--uploadForm">
-          Attach images by <span class="wc-UploadForm-label--link">selecting them</span>
-        </div>
-        <input accept=".jpe,.jpg,.jpeg,.gif,.png,.bmp"
-               class="wc-ButtonUpload js-buttonUpload"
-               id="image"
-               name="image"
-               type="file"
-        />
-      </div>
-      <img class="wc-Upload-Loader wc-Issue-Loader js-Upload-Loader"
-           src="/img/upload-loader.svg" alt="Uploading image">
-    </label>
-  </form>
-</script>
+  <div class="form-upload js-image-upload js-Form-group">
```

Comment:
Do we want `js-image-upload` here? It's not doing anthing, since the relevant view class is `js-ImageUploadView`.

```js
issues.ImageUploadView = Backbone.View.extend({
  el: $(".js-ImageUploadView"),
...
```

```html
<div class="js-ImageUploadView">
  {% include "issue/upload-image.jst" %}
</div>
```

Reply:
We do as this will show the preview of the image.
(The idea behind it is to create one module that works in the form as well as in the comments.)
https://github.com/webcompat/webcompat.com/blob/issues/2011/webcompat/static/js/lib/bugform.js#L11

This is not working properly yet, but there is a follow up issue: #2064